isthmus/build.gradle.kts

```diff
@@ -71,43 +71,126 @@ java {
-  )
```

Comment:
Extracted this out into a list which I then turn into a string for readability purposes. Also sorted the values.

Reply:
Looks much better 👏